src/web/js/check-ui.js

```diff
@@ -189,8 +189,9 @@
 
         FailingTestSkeleton.prototype.refresh = function refresh() {
           var snippets = this.tombstone.querySelectorAll(".CodeMirror");
-          for (var i = 0; i < snippets.length; i++) {
-            snippets[i].CodeMirror.refresh();
+          for (let i = 0; i < snippets.length; i++) {
+            window.requestAnimationFrame(
```

Comment:
Would it be better to hoist the request out of the loop, and refresh all the cm instances at once?